test/test_torch.py

```diff
@@ -17485,27 +17485,75 @@ def test_rdiv(self, device, dtype):
         z = torch.tensor([30 / v.item() for v in x], device=device)
         self.assertEqual(y, z, exact_dtype=False)
 
-    @onlyCPU
+    # TO DO
+    # Cross product dtypes after fix type promotion
     @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False, include_bool=False, include_complex=False))
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
Actually... do we have any tests for fmod_ ?

Reply:
Good point...